src/main/kotlin/de/amplimind/codingchallenge/controller/AdminController.kt

```diff
@@ -45,12 +39,22 @@ class AdminController(
     @ApiResponse(responseCode = "404", description = "User with email was not found.")
     @GetMapping("fetch/projects/{email}")
     fun fetchUserInfosForEmail(
-        @PathVariable email: String,
+            @PathVariable email: String,
     ): ResponseEntity<UserInfoDTO> {
         val userInfo = this.userService.fetchUserInfosForEmail(email)
         return ResponseEntity.ok(userInfo)
     }
 
+    @Operation(summary = "Endpoint for deleting a user by email")
+    @ApiResponse(responseCode = "200", description = "User was deleted successfully.")
+    @DeleteMapping("user/{email}")
```

Comment:
Ich weis nicht warum aber wenn ich das ganze in Postman teste bekomm ich immer einen 404, ich hab auch mal gedebugged, aber es wird noch nicht mal die methode im service erreicht? Kannst du das mal abchecken vllt mach ich auch was falsch^^
![image](https://github.com/amplimindcc/backend/assets/68387530/9038e73d-41da-4299-8f0c-5fc69cba48ee)

Reply:
Bei mir gehts wenn du die email nicht als query string, sondern direkt die email an sich übergibst (also zb v1/admin/user/user@web.de)